### PR TITLE
[4.0] Moved style to the CSS file (backend - system template)

### PR DESCRIPTION
**Pull request by jeckodevelopment**
### Summary of Changes
Just moved CSS rules to the error.css file in system template (backend)

### Expected result
Error page displayed properly.

### Actual result
Error page displayed properly.

### Documentation Changes Required
None

### CODE REVIEW

File: administrator/templates/system/css/error.css [hunk 1]
```diff
@@ -4,9 +4,11 @@
  */
 
 .outline {
+	margin: 0 auto; 
+	width: 550px;
 	border: 1px solid #cccccc;
 	background: #ffffff;
-	padding: 2px;
+	padding: 2px;
```
Comment (Quy) [inline administrator/templates/system/css/error.css]: Remove trailing spaces

**Comment by jeckodevelopment** [comment]
done, thanks @Quy

**Comment by wilsonge** [comment]
LGTM! Thanks!

**Comment by jeckodevelopment** [comment]
Thanks

**Pull request merged**